CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+## Katkıda bulunmadan önce
+
+Lütfen aşağıdaki maddeleri okuyup doğru olduklarından emin olduktan sonra katkıda bulununuz.
+
+- Katkı Çağrısını okudunuz mu? Okumadıysanız lütfen önce [burayı](https://github.com/fatihacet/turkcekaynaklar-com/blob/master/Katki-Cagrisi.md) okuyunuz.
+
+
+- Pull Request açmak için aşağıdakilerden birini yapabilirsiniz.
+	- Projeyi fork'layıp kendi projenizde değişiklikleri yapıp Pull Request açabilirsiniz.
+	- README.md dosyasını açıp, sağ üstteki kalem ikonuna tıklayıp istediğiniz değişikliği yapıp commit edebilirsiniz. Bu yöntemde GitHub sizin için bir patch branch'i oluşturup PR açmanıza izin vericektir.
+
+- Fork reponuz güncel mi?
+	Lütfen en son değişiklikleri kendi reponuza güncelleştiriniz.
+
+- Conflict (çakışma) var mı?
+	Eğer 'Can’t automatically merge' yazısı ile karşılaşırsanız bir çakışma var demektir. Lütfen değişikliklerinizi kontrol edip, olası conflict'leri çözünüz.
```

Comment:
Çakışma yazısını şu an için bıraktım. İleride terimler gibi bir başlık açılırsa tekrar güncellenebilir.